src/frontend/get-jira-configuration.ts

```diff
@@ -6,7 +6,7 @@ import { metricError } from "../config/metric-names";
 import { AppInstallation, FailedAppInstallation } from "../config/interfaces";
 import { GitHubAPI } from "probot";
 import Logger from "bunyan";
-import _ from "lodash";
+import { groupBy } from "lodash";
```

Comment:
🤟 Do we want to capture this preference in the conventions?
https://hello.atlassian.net/wiki/spaces/PF/pages/1424951560/Arc+Coding+Conventions

Reply:
Yeah, I think so.  Latest version of ESLint is mentioning it and it's a better standard to use.  Would be good to have it in there too.